osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -358,6 +356,10 @@ protected override void PopIn()
             this.FadeIn(transition_length, Easing.OutQuint);
 
             textbox.HoldFocus = true;
+
+            if (channelManager.CurrentChannel.Value == null || channelManager.CurrentChannel.Value is ChannelSelectorTabItem.ChannelSelectorTabChannel)
```

Comment:
Check `channelManager.AvailableChannels` has channels before showing an empty dialog (and handle the case where they become available). Maybe consider doing this in another PR (after opening an issue for that flow).

This can probably just be removed until that is done, as it's not fixing the issue this PR is addressing.